examples/postgres-v3/README.md

```diff
@@ -2,11 +2,23 @@
 
 This example shows how to access a PostgreSQL database from Spin component.
 
+## Prerequisite: Postgres
+
+This example assumes postgres is running and accessible locally via its standard 5432 port.
+
+We suggest running the `postgres` Docker container which has the necessary postgres user permissions
+already configured. For example:
+
+```
+docker run --rm -h 127.0.0.1 -p 5432:5432 -e POSTGRES_HOST_AUTH_METHOD=trust postgres
+```
+
 ## Spin up
 
-From example root:
+Then, run the following from the root of this example:
 
 ```
+export PGUSER=postgres PGHOST=localhost
```

Comment:
When I did this I was running `createdb` and `psql` inside the container.  I guess that isn't needed?

How would you feel about putting these as arguments on the commands (`-h` / `-U`) instead of environment variables?  Just thinking about Unix/Windows portability here.  I don't have a strong feeling about it, just flagging it for consideration.  (And I guess since we use `RUST_LOG=` on the build line it's moot really.)

Reply:
👍 Always appreciate vigilance on OS-portability!